src/components/DictionaryList/DictionaryListPresentation.js

```diff
@@ -111,11 +110,13 @@ function DictionaryListPresentation({
                   </tr>
                 </thead>
                 <tbody className="bg-white divide-y divide-charcoal-200">
-                  {infiniteQueryResponse?.data?.pages?.map((page) => (
+                  {infiniteQueryResponse?.data?.pages?.map((page, j) => (
```

Comment:
Not essential but to improve readability `j` could be `pageIndex` and `i` could be `resultIndex`